pkg/apis/kubic/v1beta1/registry_types_test.go

```diff
@@ -18,46 +18,44 @@
 package v1beta1
 
 import (
-	"testing"
-
-	"github.com/kubic-project/registries-operator/pkg/test"
 	"github.com/onsi/gomega"
 	"golang.org/x/net/context"
+	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/client/fake"
+	"github.com/kubic-project/registries-operator/pkg/test"
+	"testing"
 )
 
-func TestStorageRegistry(t *testing.T) {
+func TestGetCertificateFound(t *testing.T) {
+
+	g := gomega.NewGomegaWithT(t)
+
+	c := fake.NewFakeClient()
 
-	test.SkipUnlessIntegrationTesting(t)
+	r := FooRegistry;
 
-	key := types.NamespacedName{
-		Name:      "foo",
-		Namespace: "default",
+	expected, err := test.BuildSecretFromCert("foo-ca-crt","foo.crt")
+
+        if err != nil  {
+		t.Errorf("Error creating secret %v", err)
```

Comment:
thought: instead of this `if` statement maybe it could be written like `g.Expect(err).To(gomega.Equal(nil))`? I'm not sure if this is an abuse or right way. It seems if we are using gomega we should do it this way to be consistent?